.github/workflows/third_party_notices_check.yml

```diff
@@ -0,0 +1,29 @@
+name: Check Third Party Notices File
+
+on: pull_request
+
+jobs:
+  license-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+      - name: Install generate-license-file
+        run: npm install -g generate-license-file
+      - uses: actions/checkout@v2
+      - run: npm ci
+      - name: Generate license file
+        run: generate-license-file --input package.json --output ./THIRD-PARTY-NOTICES
+      - name: Git diff
+        run: git diff
+      - name: Check if there are changes
+        id: changes
```

Comment:
for my edification what is id here?

Reply:
it's a unique identifier for the step that can be to reference the step in contexts. This portion is copied from vaccine's github action, where I assumed they would have want to use that id to check if there's changes before calling the next step to commit changes (`if: steps.changes.outputs.changed == 1`). However, I looked through `EndBug/add-and-commit@v7`. Seems like if there's no changes to the targeted file, a commit can't be made anyway so this step is not necessary.